index.js

```diff
@@ -7,7 +7,7 @@ module.exports = function (obj) {
 
 function walk(obj) {
 	if (!obj || typeof obj !== 'object') { return obj; }
-	if (isDate(obj) || isRegex(obj)) { return obj; }
+	if (isDate(obj) || isRegex(obj) || Buffer.isBuffer(obj)) { return obj; }
```

Comment:
I'm concerned that referencing `Buffer` would cause an entire Buffer polyfill to be pulled in unnecessarily. I think there's other packages that have workarounds for this.

Reply:
> I'm concerned that referencing `Buffer` would cause an entire Buffer polyfill to be pulled in unnecessarily

You're right. :confused:

> I think there's other packages that have workarounds for this.

Do you have a solution in mind?

Reply:
https://github.com/ljharb/qs/blob/20820fa2dd907910de529e2564ff42aac7db6246/lib/utils.js#L218-L224 is what qs uses; I assume that would work here.

Reply:
Probably by putting two implementations in separate files - and having the browser field point to the qs one, and the normal path be the node one?

Reply:
You may be able to use the `is-buffer` package for this (but at v1, not v2)

Reply:
I will try with [`is-buffer`](https://github.com/feross/is-buffer) v1.